app/models/concerns/hud_concerns/client.rb

```diff
@@ -45,5 +45,25 @@ def gender_multi
         gm << self.GenderNone if gm.empty? && self.GenderNone.in?([8, 9, 99])
       end
     end
+
+    scope :age_group, ->(start_age: 0, end_age: nil) do
+      start_age = 0 unless start_age.is_a?(Integer)
+      end_age   = nil unless end_age.is_a?(Integer)
+      if end_age.present?
+        where(DOB: end_age.years.ago..start_age.years.ago)
+      else
+        where(arel_table[:DOB].lteq(start_age.years.ago))
+      end
```

Comment:
What you have here is totally fine, but you can now have open ranges:
 ```
GrdaWarehouse::Hud::Client.where(DOB: nil..5.years.ago).to_sql
=> "SELECT \"Client\".* FROM \"Client\" WHERE \"Client\".\"DateDeleted\" IS NULL AND \"Client\".\"DOB\" <= '2018-06-07'"
```
```suggestion

      where(DOB: end_age.&years&.ago..start_age.years.ago)
    
```